assets/js/state/databases.js

```diff
@@ -116,6 +139,8 @@ export const {
   updateDatabaseInstanceSystemReplication,
   addTagToDatabase,
   removeTagFromDatabase,
+  setDatabaseInstanceDeregistering,
+  setDatabaseInstanceNotDeregistering,
```

Comment:
Not 100% sure that I understand the reasoning behind the naming properly, but wouldn't something like unsetDatabaseInstanceDeregistering be more accurate and easier to read?

Reply:
I just replicated the naming we use in the host reducer.
In case we want to change, I think we should change all of them.
I'm not against changing anyway. Would you do it in the PR, or in a new one where I could replace all the cases?

Reply:
I just changed all of them, as it is not a big deal. It was only this way in the host reducer files